RESOURCES.md

```diff
@@ -5,25 +5,79 @@ mathsteps@socratic.org
 
 ### Existing open source step-by-step solvers
 
-[MathPiper](mathpiper.org) by [Ted Kosan](https://github.com/tkosan)
+- [PRESS PRolog Equation Solving System](https://github.com/maths/PRESS)
 
-- [Video demonstration](https://www.youtube.com/watch?v=cy6bwNBkAK0)
-- [Solver and rules](https://bitbucket.org/tkosan/mathpiperide/src/67c8b031230bc3fcd9776700487b570ebe62717b/src/library_apps/mathpiper4/src/org/mathpiper/scripts4/steps/?at=master)
- - If you would like to play with the solver and manipulator let Ted know, and
-   they will put up instructions on how to use them somewhere.
+    A computer algebra system for solving symbolic, transcendental,
+    non-differential equations. The equations in the PRESS test suite have
+    largely been taken from English A-level examination papers from the years
+    1971-1979. The system has four different top-level modules: one for solving
+    single equations, one for sets of simultaneous equations, one for
+    inequalities, and one for proving identities. The procedure for solving
+    single equations is the central core of the program. The other top-level
+    modules are largely interfaces to the relevant parts of the single equation
+    code. (See below for more information on PRESS.)
 
+- [MathPiper](http://mathpiper.org)
+    
+    A conventional computer algebra system that has been modified to support
+    step-by-step mathematics. The step-by-step equation solver in MathPiper
+    (which is based on PRESS) has not been officially released yet, so if you
+    would like to experiment with it contact [Ted
+    Kosan](https://github.com/tkosan). 
```

Comment:
Yes, PRESS is a step-by-step solver. Here is an example of its output:

?- solve(log(2,x) + 4*log(x,2) = 5).
Solving log(2, x) + 4 * log(x, 2) = 5 for x
Offending set is [log(2, x), log(x, 2)]
Reduced term is _G391
Rewriting equation in terms of log(2, x)
gives log(2, x) + 4 * log(2, x) ^ -1 = 5
Substituting   x1 for log(2, x) gives
 4 * x1 ^ -1 + x1 = 5
Multiply through by x1 to get 
x1 ^ 2 + -5 * x1 + 4 = 0
Using quadratic equation formula
Solutions are x1 = 4 and x1 = 1
Applying substitution 
    x1 = log(2, x)
   to    : 
    x1 = 4 # x1 = 1
   gives : 
    log(2, x) = 4 # log(2, x) = 1
Solving disjunct log(2, x) = 4
    x = 16
     (by Isolation)
Solving disjunct log(2, x) = 1
    x = 2
     (by Isolation)
Answer is : 
(X1 # X2)
  where :
    X1 =  x = 16
    X2 =  x = 2